prefs.js

```diff
@@ -148,9 +145,12 @@ export default class GoogleEarthWallpaperPrefs extends ExtensionPreferences {
         }
 
         settings.bind('delete-previous', deleteSwitch, 'active', Gio.SettingsBindFlags.DEFAULT);
-        const intrvs = interval_names.map(_);
+
+        const intervals = [ 300, 600, 1800, 3600, 4800, 21600, 86400 ];
+        const interval_names = [_("5 m"), _("10 m"), _("30 m"), _("60 m"), _("90 m"), _("6 h"), _("daily")];
+
```

Comment:
er, a local change `6 h` snuck in. I'll fix that (or maybe some users would like 6/12h updates??)